src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,19 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList = ({ comments }) => {
+  return comments.length === 0 ? (
+    <b data-cy="NoCommentsMessage">No comments yet</b>
+  ) : (
+    <div className="CommentList">
+      {comments.map(comment => {
+        return (
+          <div className="CommentInfo" key={comment.id}>
```

Comment:
you can move this div inside the CommentInfo and replace the fragment in that component with this tag